Jenkinsfile

```diff
@@ -9,6 +9,12 @@ node('linux && docker') {
         sh 'npm run setup'
       }
 
+      if (isMaster) {
```

Comment:
The version isn't dynamically read by headless nor atomic, it's added at build time. Therefor, headless and atomic must be built before the version is bumped.